package.json

```diff
@@ -97,7 +97,7 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "factoria": "^3.2.2",
     "faker": "^5.5.3",
-    "flow-bin": "^0.162.0",
+    "flow-bin": "^0.185.0",
```

Comment:
No objection, but any advantage to this version bump?

Reply:
I was hitting a bug locally that was fixed in 0.185.0:

> Fix a bug that makes assignments in loop guards like x = some expression containing x have a type that's effectively any. If you turned on inference_mode=constrain_writes since the last release, there will be new errors.

https://github.com/facebook/flow/blob/main/Changelog.md

Reply:
Cool, thanks. FWIW, it will help me review future PRs if you can mention things like this in your commit messages or the PR body.